cmd/device-definitions-api/populate_device_features.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"strings"
+
+	"github.com/DIMO-Network/device-definitions-api/internal/config"
+	"github.com/DIMO-Network/device-definitions-api/internal/infrastructure/db/models"
+	elastic "github.com/DIMO-Network/device-definitions-api/internal/infrastructure/elastic_search"
+	"github.com/DIMO-Network/shared/db"
+	"github.com/rs/zerolog"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+)
+
+func prepareFeatureData(f map[string]map[string]int) []models.DeviceIntegrationFeatures {
+	ft := make([]models.DeviceIntegrationFeatures, len(f))
+
+	for k, v := range f {
+		feat := models.DeviceIntegrationFeatures{}
+		feat.ElasticProperty = k
+		feat.SupportLevel = 0
+
+		if v["doc_count"] > 0 {
+			feat.SupportLevel = 2
+		}
+
+		ft = append(ft, feat)
+	}
+
+	return ft
+}
+
+func populateDeviceFeaturesFromEs(ctx context.Context, logger zerolog.Logger, s *config.Settings) error {
+	pdb := db.NewDbConnectionFromSettings(ctx, &s.DB, true)
+	pdb.WaitForDB(logger)
+
+	es, _ := elastic.NewElasticSearch(s, logger)
+
+	resp, err := es.GetDeviceFeatures(s.Environment)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range resp.Aggregations.Features.Buckets {
+		intID := strings.Split(f.Key, "/")[2]
```

Comment:
In the past I've done `intID := strings.TrimPrefix(f.Key, "dimo/integration/")`.